### PR TITLE
Vault door access

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request
The mining door from engineering was broken so I mapped it at the request of the Vault LM, Ody. Engineering can do mining now I suppose.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog
:cl:
tweak: tweaked a few things
/:cl:

### CODE REVIEW

File: _maps/map_files/Blythe-small/blythe-lower.dmm [hunk 1]
```diff
@@ -6514,7 +6514,7 @@
 /area/f13/citycaves)
 "fnH" = (
 /obj/machinery/door/airlock/mining{
-	req_access_txt = "126"
```
Comment (Zero-Percent-Angel) [inline _maps/map_files/Blythe-small/blythe-lower.dmm:6517]: I set it to this specifically because anyone could get in the Vault

Comment (Zero-Percent-Angel) [reply _maps/map_files/Blythe-small/blythe-lower.dmm:6517]: We don't really want random wasties being able to walk in through the Vault mine

Comment (AllCorruptingWyrm) [reply _maps/map_files/Blythe-small/blythe-lower.dmm:6517]: oh my bad. I'll set access, engineers couldn't go through the mines.

Comment (Zero-Percent-Angel) [reply _maps/map_files/Blythe-small/blythe-lower.dmm]: It's alright, I'm fixing the role in the code :)

**Pull request merged**